dev/fattest.simplicity/src/componenttest/containers/ExternalTestServiceDockerClientStrategy.java

```diff
@@ -302,15 +308,65 @@ protected boolean isApplicable() {
         return true;
     }
 
-    public static boolean useRemoteDocker() {
-        if (USE_REMOTE_DOCKER) {
-            return true; // remote docker explicitly requested
-        }
-        if (Boolean.parseBoolean(System.getenv("GITHUB_ACTIONS"))) {
-            return false; // always use local docker for GH Actions
-        }
-        // Otherwise, use local docker for local runs, and remote docker for remote (RTC) runs
-        return !FATRunner.FAT_TEST_LOCALRUN;
-    }
+    /**
+     * Determines if we are going to attempt to run against a remote
+     * docker host, or a local docker host.
+     *
+     * Priority:
+     * 1. System Property: fat.test.use.remote.docker
+     * 2. System Property: fat.test.docker.host -> REMOTE
+     * 3. System: GITHUB_ACTIONS -> LOCAL
+     * 4. System: WINDOWS -> REMOTE
+     *
+     * default (!!! fat.test.localrun)
+     *
+     * @return true, we are running against a remote docker host, false otherwise.
+     */
+    private static boolean useRemoteDocker() {
+        boolean result;
+        String reason;
+
+        do {
+            //State 1: fat.test.use.remote.docker should always be honored first
+            if (System.getProperty("fat.test.use.remote.docker") != null) {
+                result = Boolean.getBoolean("fat.test.use.remote.docker");
+                reason = "fat.test.use.remote.docker set to " + result;
+                break;
```

Comment:
What if a "fat.test.use.remote.docker" is true, but "fat.test.docker.host" is blank/null?

Reply:
fat.test.use.remote.docker will make the priority of ExternalTestServiceDockerClientStategy (this class) high (900) so it will be the first strategy that docker-java (the underlying API) will use to try to connect to a docker host. 

This class uses user.build.properties on the machines home directory to get a list of possible hosts to try.  If fat.test.docker.host is set, it will ignore all but the host that matches that one. 

fat.test.docker.host is really meant so that if a certain docker host is flaky we can explicitly avoid or test against it.